src/mplhep/plot.py

```diff
@@ -392,17 +429,45 @@ def iterable_not_string(arg):
 
     # Add sticky edges for autoscale
     _artist.sticky_edges.y.append(0)
-
     if xtick_labels is None:
         if binticks:
             _slice = int(round(float(len(final_bins)) / len(ax.get_xticks()))) + 1
             ax.set_xticks(final_bins[::_slice])
+    elif flow == "show":
+        if binticks:
+            _slice = int(round(float(len(final_bins)) / len(ax.get_xticks()))) + 1
+            ax.set_xticks(final_bins[::_slice])
     else:
         ax.set_xticks(_bin_centers)
         ax.set_xticklabels(xtick_labels)
 
     if x_axes_label:
         ax.set_xlabel(x_axes_label)
+    if flow == "hint":
+        underflow, overflow = 0.0, 0.0
+        for h in hists:
+            underflow = underflow + h.view(flow=True)["value"][0]
```

Comment:
Is the intended behaviour for a stack of histogram a single flow bin?

Reply:
yes, it's intended to just show stacked number, showing sum of the stacked bin

Reply:
If it's the `show` option, it stacked like other bins.

Reply:
add new test plots 
<img width="263" alt="image" src="https://user-images.githubusercontent.com/20898477/226146263-cf24e057-5b58-4a12-84df-acd4d36445f3.png">